src/transit.js

```diff
@@ -420,7 +421,8 @@ class Transit {
 			this.logger.warn(
 				`Incoming '${payload.event}' event from '${payload.sender}' node is dropped, because broker is stopped.`
 			);
-			return;
+			// return false so the transporter knows this event wasn't handled.
+			return this.Promise.reject(false);
```

Comment:
Please change it because rejecting non `Error` can cause problems. Every place in the project the catch always got an `Error`. Returning `this.Promise.resolve(false)` can be good?

Reply:
Made this change.
eventHandler now resolves true if the message was handled, or false in the situation where the broker is stopped